ParseException when large repository size

There are problems with large repositories? 
`2018-09-13 06:13:56,450 ERROR [qtp2049582890-454367] w.b.w.c.e.m.DefaultExceptionMapper [DefaultExceptionMapper.java:21] Something went wrong.
com.heroku.api.exception.ParseException: Failed to parse JSON:[{"acm":false,"archived_at":null,"buildpack_provided_description":"PHP (HHVM)","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2013-09-13T21:19:36Z","id":"9e71b4ac-2f6b-41fe-9757-ec1337b3ff6e","git_url":"https://git.heroku.com/appolo-hhvm.git","maintenance":false,"name":"appolo-hhvm","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2013-09-13T22:01:20Z","repo_size":149245952,"slug_size":125913309,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T13:17:22Z","web_url":"https://appolo-hhvm.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2016-07-28T11:56:47Z","id":"f339595c-e608-4be9-a919-3e7e042aae00","git_url":"https://git.heroku.com/ardene-bot.git","maintenance":false,"name":"ardene-bot","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-12-04T15:55:08Z","repo_size":170226,"slug_size":21939436,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2016-12-04T15:55:16Z","web_url":"https://ardene-bot.herokuapp.com/"},{"acm":true,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2017-11-09T15:07:18Z","id":"df391958-e07b-4433-a74f-dd7f2f3a0fc8","git_url":"https://git.heroku.com/ardene-coupon-api-prod.git","maintenance":false,"name":"ardene-coupon-api-prod","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-05-02T20:09:27Z","repo_size":700883,"slug_size":81231738,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-09-10T14:04:26Z","web_url":"https://ardene-coupon-api-prod.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"PHP","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2014-02-21T18:38:04Z","id":"49d49cb2-23a0-42ea-af5b-f43b7117077c","git_url":"https://git.heroku.com/bellmedia-dev.git","maintenance":false,"name":"bellmedia-dev","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2018-05-23T20:16:40Z","repo_size":3535992526,"slug_size":226173364,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2018-05-23T20:16:40Z","web_url":"https://bellmedia-dev.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Multipack","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2014-03-18T20:04:48Z","id":"75aa9f8f-64da-4fed-a157-761853972b5e","git_url":"https://git.heroku.com/bellmedia-prod.git","maintenance":false,"name":"bellmedia-prod","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-03-30T14:23:41Z","repo_size":212354913,"slug_size":224175063,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2017-12-06T21:03:08Z","web_url":"https://bellmedia-prod.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"PHP","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2014-02-27T19:50:43Z","id":"500c375f-2f20-4145-a914-baf410fcf80e","git_url":"https://git.heroku.com/bellmedia-staging.git","maintenance":false,"name":"bellmedia-staging","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2015-11-13T14:15:44Z","repo_size":455004058,"slug_size":240800238,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2017-12-06T20:29:17Z","web_url":"https://bellmedia-staging.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"PHP (composer.json)","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2014-05-21T19:20:48Z","id":"1007c276-0481-4057-aec7-2961054eb294","git_url":"https://git.heroku.com/bellmedia-upfront.git","maintenance":false,"name":"bellmedia-upfront","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2015-09-18T17:53:55Z","repo_size":561167100,"slug_size":166056112,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T23:00:12Z","web_url":"https://bellmedia-upfront.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Rails","build_stack":{"id":"3713e6e6-3de7-4224-9bc1-b0f515415de7","name":"bamboo-ree-1.8.7"},"created_at":"2011-06-02T00:31:50Z","id":"cfbf15a8-c4da-40e8-bdac-81847bfd155c","git_url":"https://git.heroku.com/coupond.git","maintenance":false,"name":"coupond","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-05-10T04:48:19Z","repo_size":17039360,"slug_size":13893632,"stack":{"id":"3713e6e6-3de7-4224-9bc1-b0f515415de7","name":"bamboo-ree-1.8.7"},"updated_at":"2016-05-10T04:48:19Z","web_url":"https://coupond.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2016-11-07T18:30:49Z","id":"0c193daf-1b06-4c2b-a18f-478a34b8de84","git_url":"https://git.heroku.com/degama-dtms-proxy.git","maintenance":false,"name":"degama-dtms-proxy","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-11-07T20:48:31Z","repo_size":17142,"slug_size":17560479,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2016-11-07T21:27:47Z","web_url":"https://degama-dtms-proxy.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2018-05-17T18:27:39Z","id":"6b642db0-b8d2-4041-ba33-712867f1e4bd","git_url":"https://git.heroku.com/dinerenblanc-api-staging.git","maintenance":false,"name":"dinerenblanc-api-staging","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-06-01T00:33:05Z","repo_size":null,"slug_size":48539145,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-06-01T00:33:05Z","web_url":"https://dinerenblanc-api-staging.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2018-05-17T18:50:19Z","id":"cf92d9f8-3d73-431b-881e-bbf30157f99c","git_url":"https://git.heroku.com/dinerenblanc-proxy-staging.git","maintenance":false,"name":"dinerenblanc-proxy-staging","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-06-01T00:33:03Z","repo_size":null,"slug_size":48563037,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-06-01T00:33:03Z","web_url":"https://dinerenblanc-proxy-staging.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":null,"build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2016-03-17T18:52:18Z","id":"d2531639-9f43-45fc-966d-1cfa3cd371d6","git_url":"https://git.heroku.com/dtms-proxy.git","maintenance":false,"name":"dtms-proxy","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-03-17T18:52:19Z","repo_size":null,"slug_size":null,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2016-03-17T18:53:25Z","web_url":"https://dtms-proxy.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2017-01-20T17:54:23Z","id":"e06c116d-0ea5-46ea-84c4-fe41bc74bd03","git_url":"https://git.heroku.com/duology-flow.git","maintenance":false,"name":"duology-flow","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2017-05-03T21:15:14Z","repo_size":null,"slug_size":94524105,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2017-05-03T21:15:15Z","web_url":"https://duology-flow.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Ruby/Rails","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2012-12-06T00:07:28Z","id":"60777d3b-c1dd-4c82-9093-16001fd8f1e5","git_url":"https://git.heroku.com/engagebox-astral.git","maintenance":false,"name":"engagebox-astral","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-05-23T12:23:25Z","repo_size":43958272,"slug_size":35860480,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T02:42:50Z","web_url":"https://engagebox-astral.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":null,"build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2012-06-22T14:51:16Z","id":"701017a8-1209-4242-81cb-ca9d8f039947","git_url":"https://git.heroku.com/engagebox-david.git","maintenance":false,"name":"engagebox-david","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2012-09-19T20:39:25Z","repo_size":null,"slug_size":null,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2017-12-05T23:15:32Z","web_url":"https://engagebox-david.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Ruby","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2012-10-11T15:43:52Z","id":"047c122d-362f-4a9e-bd00-95e7c73b0292","git_url":"https://git.heroku.com/engagebox-development.git","maintenance":false,"name":"engagebox-development","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-05-21T10:36:12Z","repo_size":43335680,"slug_size":97846998,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T01:20:17Z","web_url":"https://engagebox-development.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Ruby","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2012-10-11T15:42:35Z","id":"08f8a1a4-e795-4cd9-a777-10a6ccf432fa","git_url":"https://git.heroku.com/engagebox-master.git","maintenance":false,"name":"engagebox-master","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2015-07-25T17:34:07Z","repo_size":41463808,"slug_size":97805882,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T01:20:17Z","web_url":"https://engagebox-master.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Ruby/Rails","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2012-04-11T16:07:29Z","id":"c0af47cb-afe7-46b0-8e95-179ab6dfd749","git_url":"https://git.heroku.com/engagebox-old.git","maintenance":false,"name":"engagebox-old","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-02-09T20:59:13Z","repo_size":50745344,"slug_size":35688448,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-05T21:54:45Z","web_url":"https://engagebox-old.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Ruby","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2013-07-19T15:24:29Z","id":"d9eb130a-2866-4972-81a9-6e0fcbeb3745","git_url":"https://git.heroku.com/engagebox-staging.git","maintenance":false,"name":"engagebox-staging","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2015-11-27T20:21:35Z","repo_size":62668800,"slug_size":97805882,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T09:38:16Z","web_url":"https://engagebox-staging.herokuapp.com/"},{"acm":false,"archived_at":"2013-08-20T21:08:26Z","buildpack_provided_description":"Ruby/Rack","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2012-06-20T03:38:15Z","id":"fbc06dd6-1acf-4bd7-8fc8-eb7bb29b3ca8","git_url":"https://git.heroku.com/falling-leaf-1374.git","maintenance":false,"name":"falling-leaf-1374","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2012-06-20T03:38:18Z","repo_size":null,"slug_size":null,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2015-04-20T22:00:58Z","web_url":"https://falling-leaf-1374.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"PHP","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2013-01-02T19:01:25Z","id":"ab33caa3-d14d-48d5-8082-3ea6ef8427ec","git_url":"https://git.heroku.com/grille-canalvie.git","maintenance":false,"name":"grille-canalvie","owner":{"email":"samuel.halle@okam.ca","id":"044f0cb4-bbf5-463a-874e-72db73c903eb"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2013-03-15T14:19:39Z","repo_size":96641024,"slug_size":104718336,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T03:17:44Z","web_url":"https://grille-canalvie.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"PHP","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2012-12-23T22:35:01Z","id":"c28bc248-43f7-49e6-9703-ec4707ab747a","git_url":"https://git.heroku.com/grille-seriesplus.git","maintenance":false,"name":"grille-seriesplus","owner":{"email":"samuel.halle@okam.ca","id":"044f0cb4-bbf5-463a-874e-72db73c903eb"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2013-01-21T16:15:23Z","repo_size":43421696,"slug_size":51511296,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T03:07:28Z","web_url":"https://grille-seriesplus.herokuapp.com/"},{"acm":true,"archived_at":null,"buildpack_provided_description":"Git Deploy Keys,Node.js","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2017-08-22T23:26:45Z","id":"64898960-0a32-49c0-8206-c8a1ab39aec4","git_url":"https://git.heroku.com/hisports-api-prod.git","maintenance":false,"name":"hisports-api-prod","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-05-31T17:56:17Z","repo_size":125213,"slug_size":45711843,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-05-31T17:56:17Z","web_url":"https://hisports-api-prod.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Git Deploy Keys,Node.js","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2017-07-21T18:47:38Z","id":"d89c33ae-be60-44a2-af04-e691e1e03d95","git_url":"https://git.heroku.com/hisports-api-staging.git","maintenance":false,"name":"hisports-api-staging","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-05-02T14:35:53Z","repo_size":296784,"slug_size":45711843,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-05-02T14:36:53Z","web_url":"https://hisports-api-staging.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Git Deploy Keys,Node.js","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2017-10-30T19:18:55Z","id":"7e6e9448-42e4-4a88-beca-96db68043d7d","git_url":"https://git.heroku.com/hisports-apps.git","maintenance":false,"name":"hisports-apps","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2017-10-30T20:24:22Z","repo_size":null,"slug_size":104952968,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-01-21T00:38:31Z","web_url":"https://hisports-apps.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2018-06-22T14:18:19Z","id":"22cb70d7-571b-4001-8664-ea44e983962b","git_url":"https://git.heroku.com/hisports-next.git","maintenance":false,"name":"hisports-next","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-07-03T15:11:46Z","repo_size":41294747,"slug_size":264404995,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-07-03T15:12:48Z","web_url":"https://hisports-next.herokuapp.com/"},{"acm":true,"archived_at":null,"buildpack_provided_description":"Google Chrome,Node.js,Fonts","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2017-10-17T01:02:33Z","id":"064013e7-3ecc-40ab-8b01-d72c67c660bd","git_url":"https://git.heroku.com/hisports-pdf.git","maintenance":false,"name":"hisports-pdf","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-03-26T16:10:12Z","repo_size":311582,"slug_size":371273161,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-05-15T00:08:02Z","web_url":"https://hisports-pdf.herokuapp.com/"},{"acm":true,"archived_at":null,"buildpack_provided_description":"Google Chrome,Node.js,Fonts","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2018-07-27T20:45:06Z","id":"f7231f7f-fbdf-4d5d-9a57-4363b7554bdb","git_url":"https://git.heroku.com/hisports-reports.git","maintenance":false,"name":"hisports-reports","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-08-03T16:05:38Z","repo_size":302967,"slug_size":327820928,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-08-03T16:06:38Z","web_url":"https://hisports-reports.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Git Deploy Keys,Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2017-09-07T21:28:21Z","id":"008c86a9-97d7-4fed-8812-7a04c6400ceb","git_url":"https://git.heroku.com/hisports-web-prod.git","maintenance":false,"name":"hisports-web-prod","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-05-30T21:47:19Z","repo_size":null,"slug_size":126966344,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2018-05-30T21:47:19Z","web_url":"https://hisports-web-prod.herokuapp.com/"},{"acm":true,"archived_at":null,"buildpack_provided_description":"Git Deploy Keys,Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2015-10-29T23:46:58Z","id":"df697e04-3d49-4815-8123-2a6985882a0e","git_url":"https://git.heroku.com/hisports-web-staging.git","maintenance":false,"name":"hisports-web-staging","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-04-04T21:10:17Z","repo_size":2564007,"slug_size":129103754,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2018-05-14T19:51:35Z","web_url":"https://hisports-web-staging.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":null,"build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2018-09-12T21:08:01Z","id":"e50c8283-e057-42f2-9b79-8eb2abf814f6","git_url":"https://git.heroku.com/keolis-bot-staging.git","maintenance":false,"name":"keolis-bot-staging","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-09-12T21:08:01Z","repo_size":null,"slug_size":null,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-09-12T21:08:01Z","web_url":"https://keolis-bot-staging.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2018-09-05T20:06:03Z","id":"79f711e5-bba0-40ed-91dc-bb4e5a76dec9","git_url":"https://git.heroku.com/keolis-chatbot.git","maintenance":false,"name":"keolis-chatbot","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-09-12T20:27:25Z","repo_size":234320,"slug_size":74180308,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-09-12T20:28:25Z","web_url":"https://keolis-chatbot.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Ruby/Rails","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2012-08-28T15:53:56Z","id":"d0bfd7aa-345f-4560-95d4-5da31e4551cc","git_url":"https://git.heroku.com/kioske.git","maintenance":false,"name":"kioske","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2015-12-01T15:45:31Z","repo_size":49016832,"slug_size":51789824,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T00:29:17Z","web_url":"https://kioske.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Ruby/Rails","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2013-08-05T17:34:50Z","id":"5b9d86c0-88d9-47f6-b742-89e0091a0229","git_url":"https://git.heroku.com/loco-prod.git","maintenance":false,"name":"loco-prod","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-12-16T15:43:53Z","repo_size":26951680,"slug_size":66501394,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T10:44:20Z","web_url":"https://loco-prod.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"PHP (composer.json)","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2014-02-24T20:00:28Z","id":"2707d686-4cdd-4c7c-8fdb-6ab515ab1288","git_url":"https://git.heroku.com/mtlarabais-staging.git","maintenance":false,"name":"mtlarabais-staging","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2015-09-18T18:22:09Z","repo_size":64114688,"slug_size":97119462,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T20:23:06Z","web_url":"https://mtlarabais-staging.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"PHP","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2012-02-29T17:21:29Z","id":"6702509f-9b25-4614-82c0-a7cc18a34409","git_url":"https://git.heroku.com/okamcontest.git","maintenance":false,"name":"okamcontest","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2012-11-27T17:14:57Z","repo_size":82919424,"slug_size":22896640,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-05T21:06:32Z","web_url":"https://okamcontest.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Ruby","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2015-01-27T21:35:50Z","id":"722bad8a-af54-409d-968e-f865b1e13e29","git_url":"https://git.heroku.com/okam-dashboard.git","maintenance":false,"name":"okam-dashboard","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2015-01-28T13:31:50Z","repo_size":417818,"slug_size":26905082,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2017-12-07T06:14:24Z","web_url":"https://okam-dashboard.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2014-03-04T23:47:00Z","id":"5fb4ef26-e415-40bb-9ad9-f06bf12309f4","git_url":"https://git.heroku.com/okam-hubot.git","maintenance":false,"name":"okam-hubot","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2018-06-01T21:26:21Z","repo_size":123400,"slug_size":17810106,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2018-06-01T21:26:21Z","web_url":"https://okam-hubot.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Meteor app","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2013-05-07T20:51:05Z","id":"24d5e502-2646-4041-83c2-1057ea56f50c","git_url":"https://git.heroku.com/okam-land.git","maintenance":false,"name":"okam-land","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2017-03-20T19:29:54Z","repo_size":1806336,"slug_size":34926592,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T06:41:39Z","web_url":"https://okam-land.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Go","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2016-02-17T21:51:05Z","id":"93db6dc8-34ea-4125-a311-bb3759bd8c84","git_url":"https://git.heroku.com/okam-notable.git","maintenance":false,"name":"okam-notable","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2018-05-24T06:37:00Z","repo_size":151442,"slug_size":5242809,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2018-05-24T06:37:00Z","web_url":"https://okam-notable.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2016-06-30T21:12:07Z","id":"9a9e1cc3-09dd-4466-b005-4916eff2e831","git_url":"https://git.heroku.com/okamobot.git","maintenance":false,"name":"okamobot","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-08-24T16:56:33Z","repo_size":514354,"slug_size":20186177,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2016-08-24T16:57:33Z","web_url":"https://okamobot.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Ruby","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2018-01-30T15:25:07Z","id":"5905c5c1-f005-4d88-bb7a-1680ef524915","git_url":"https://git.heroku.com/okam-room.git","maintenance":false,"name":"okam-room","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2018-01-30T15:26:47Z","repo_size":null,"slug_size":39468499,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-01-30T15:26:48Z","web_url":"https://okam-room.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"created_at":"2013-04-17T13:58:06Z","id":"61a87065-ac43-4e77-b6ae-228942169881","git_url":"https://git.heroku.com/okam-weinre.git","maintenance":false,"name":"okam-weinre","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2013-04-17T13:58:41Z","repo_size":212992,"slug_size":9019392,"stack":{"id":"7e04461d-ec81-4bdd-8b37-b69b320a9f83","name":"cedar"},"updated_at":"2017-12-06T06:07:27Z","web_url":"https://okam-weinre.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2016-07-28T16:01:12Z","id":"0eb0fa60-d09b-419a-9e03-bc4b8e76b2f8","git_url":"https://git.heroku.com/papio-bot.git","maintenance":false,"name":"papio-bot","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2017-07-10T21:44:50Z","repo_size":478594,"slug_size":28296074,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2018-07-17T19:45:09Z","web_url":"https://papio-bot.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2016-08-29T20:40:00Z","id":"2945a5d2-b214-4f89-80f9-182daa13cd88","git_url":"https://git.heroku.com/papio-bot-qa.git","maintenance":false,"name":"papio-bot-qa","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2017-07-10T21:44:36Z","repo_size":318496,"slug_size":28296074,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2018-05-02T14:44:42Z","web_url":"https://papio-bot-qa.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2016-09-16T18:48:03Z","id":"67784c43-464b-4880-b9e2-f34b6c18a86f","git_url":"https://git.heroku.com/papio-invite.git","maintenance":false,"name":"papio-invite","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-09-16T19:22:16Z","repo_size":359520,"slug_size":14544152,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2016-09-16T19:23:17Z","web_url":"https://papio-invite.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":null,"build_stack":{"id":"fa15409e-a5da-4998-9954-0e865cb1a19f","name":"container"},"created_at":"2018-07-18T20:16:30Z","id":"24297304-9fc1-4519-af87-8fe1c44d2a62","git_url":"https://git.heroku.com/pssprt-prototype.git","maintenance":false,"name":"pssprt-prototype","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2018-07-18T20:19:45Z","repo_size":null,"slug_size":null,"stack":{"id":"fa15409e-a5da-4998-9954-0e865cb1a19f","name":"container"},"updated_at":"2018-07-18T20:19:45Z","web_url":"https://pssprt-prototype.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"created_at":"2017-04-03T13:47:31Z","id":"a514a043-6cee-4933-b83a-10c0084744c3","git_url":"https://git.heroku.com/saq-duology.git","maintenance":false,"name":"saq-duology","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2017-05-03T21:14:53Z","repo_size":null,"slug_size":94412683,"stack":{"id":"f9f9cbd7-2970-41ef-8db5-3df7123b041f","name":"cedar-14"},"updated_at":"2017-05-03T21:14:53Z","web_url":"https://saq-duology.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":null,"build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2017-11-09T15:06:28Z","id":"7cb4ad3d-0df8-4cec-934c-91278ad127ab","git_url":"https://git.heroku.com/thawing-taiga-28064.git","maintenance":false,"name":"thawing-taiga-28064","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2017-11-09T15:06:28Z","repo_size":null,"slug_size":null,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2017-11-09T15:06:28Z","web_url":"https://thawing-taiga-28064.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"Node.js","build_stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"created_at":"2018-03-15T18:03:04Z","id":"d76f0e5c-79c4-42ce-a79d-9a74f4cba2db","git_url":"https://git.heroku.com/ubisoft-hr.git","maintenance":false,"name":"ubisoft-hr","owner":{"email":"okamlabs@herokumanager.com","id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"team":{"id":"0a60a3cc-14c4-48ce-8005-159f5e5a0d31","name":"okamlabs"},"space":null,"internal_routing":null,"released_at":"2018-04-06T14:25:46Z","repo_size":93525,"slug_size":56335024,"stack":{"id":"ee582d3c-717d-4a57-ba5f-8b3a39f3a817","name":"heroku-16"},"updated_at":"2018-04-06T14:25:46Z","web_url":"https://ubisoft-hr.herokuapp.com/"},{"acm":false,"archived_at":null,"buildpack_provided_description":"rails","build_stack":{"id":"3713e6e6-3de7-4224-9bc1-b0f515415de7","name":"bamboo-ree-1.8.7"},"created_at":"2010-11-22T22:49:14Z","id":"f2a95574-6b3d-40a5-9f84-20b2cd423a88","git_url":"https://git.heroku.com/whosturn.git","maintenance":false,"name":"whosturn","owner":{"email":"dev@okam.ca","id":"b723fb3f-475d-4d5d-a2a9-30f256adb384"},"region":{"id":"59accabd-516d-4f0e-83e6-6e3757701145","name":"us"},"organization":null,"team":null,"space":null,"internal_routing":null,"released_at":"2016-05-05T17:47:21Z","repo_size":24645632,"slug_size":21573632,"stack":{"id":"3713e6e6-3de7-4224-9bc1-b0f515415de7","name":"bamboo-ree-1.8.7"},"updated_at":"2016-05-05T17:47:21Z","web_url":"https://whosturn.herokuapp.com/"}]
 at com.heroku.api.parser.Json.parse(Json.java:78)
 at com.heroku.api.request.app.AppList.getResponse(AppList.java:73)
 at com.heroku.api.request.app.AppList.getResponse(AppList.java:22)
 at com.heroku.api.connection.HttpClientConnection.execute(HttpClientConnection.java:99)
 at com.heroku.api.connection.HttpClientConnection.execute(HttpClientConnection.java:67)
 at com.heroku.api.HerokuAPI.listApps(HerokuAPI.java:131)
 at works.buddy.ws.integration.services.DefaultHerokuClient.getApps(DefaultHerokuClient.java:63)
 at works.buddy.ws.integration.services.project.HerokuProjectSupplier.getProjects(HerokuProjectSupplier.java:57)
 at works.buddy.ws.integration.services.project.HerokuProjectSupplier.getUserRepositories(HerokuProjectSupplier.java:33)
 at works.buddy.ws.integration.services.DefaultIntegrationService.getExternalProjects(DefaultIntegrationService.java:381)
 at works.buddy.application.internalapi.DefaultInternalAPI.getExternalProjects(DefaultInternalAPI.java:2025)
 at sun.reflect.GeneratedMethodAccessor762.invoke(Unknown Source)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:498)
 at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:343)
 at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:197)
 at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
 at org.springframework.aop.framework.adapter.MethodBeforeAdviceInterceptor.invoke(MethodBeforeAdviceInterceptor.java:52)
 at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:174)
 at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:100)
 at works.buddy.application.internalapi.aop.LoggingAspect.loggingExecutionTime(LoggingAspect.java:22)
 at sun.reflect.GeneratedMethodAccessor307.invoke(Unknown Source)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:498)
 at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:644)
 at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:633)
 at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
 at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:185)
 at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
 at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:185)
 at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212)
 at com.sun.proxy.$Proxy250.getExternalProjects(Unknown Source)
 at sun.reflect.GeneratedMethodAccessor762.invoke(Unknown Source)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:498)
 at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:179)
 at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96)
 at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:192)
 at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:103)
 at org.apache.cxf.interceptor.ServiceInvokerInterceptor$1.run(ServiceInvokerInterceptor.java:59)
 at org.apache.cxf.interceptor.ServiceInvokerInterceptor.handleMessage(ServiceInvokerInterceptor.java:96)
 at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308)
 at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121)
 at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:267)
 at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234)
 at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208)
 at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160)
 at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:216)
 at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:301)
 at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doGet(AbstractHTTPServlet.java:225)
 at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
 at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:276)
 at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:841)
 at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1650)
 at ch.qos.logback.classic.helpers.MDCInsertingServletFilter.doFilter(MDCInsertingServletFilter.java:49)
 at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
 at org.springframework.orm.hibernate5.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:152)
 at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
 at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
 at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:533)
 at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188)
 at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1595)
 at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188)
 at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1253)
 at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168)
 at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:473)
 at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1564)
 at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166)
 at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1155)
 at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
 at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
 at org.eclipse.jetty.server.Server.handle(Server.java:564)
 at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:317)
 at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251)
 at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
 at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:110)
 at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
 at org.eclipse.jetty.util.thread.Invocable.invokePreferred(Invocable.java:128)
 at org.eclipse.jetty.util.thread.Invocable$InvocableExecutor.invoke(Invocable.java:222)
 at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:294)
 at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:199)
 at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:673)
 at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:591)
 at java.lang.Thread.run(Thread.java:748)
Caused by: com.heroku.api.exception.ParseException: Unable to parse data.
 at com.heroku.api.parser.JacksonParser.parse(JacksonParser.java:40)
 at com.heroku.api.parser.Json.parse(Json.java:75)
 ... 83 common frames omitted
Caused by: org.codehaus.jackson.JsonParseException: Numeric value (3535992526) out of range of int
 at [Source: [B@5b5ec5f0; line: 1, column: 3073]
 at org.codehaus.jackson.JsonParser._constructError(JsonParser.java:1433)
 at org.codehaus.jackson.impl.JsonParserMinimalBase._reportError(JsonParserMinimalBase.java:521)
 at org.codehaus.jackson.impl.JsonParserBase.convertNumberToInt(JsonParserBase.java:829)
 at org.codehaus.jackson.impl.JsonParserBase.getIntValue(JsonParserBase.java:625)
 at org.codehaus.jackson.map.deser.std.StdDeserializer._parseInteger(StdDeserializer.java:303)
 at org.codehaus.jackson.map.deser.std.StdDeserializer$IntegerDeserializer.deserialize(StdDeserializer.java:819)
 at org.codehaus.jackson.map.deser.std.StdDeserializer$IntegerDeserializer.deserialize(StdDeserializer.java:806)
 at org.codehaus.jackson.map.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:299)
 at org.codehaus.jackson.map.deser.SettableBeanProperty$MethodProperty.deserializeAndSet(SettableBeanProperty.java:414)
 at org.codehaus.jackson.map.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:697)
 at org.codehaus.jackson.map.deser.BeanDeserializer.deserialize(BeanDeserializer.java:580)
 at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:217)
 at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:194)
 at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:30)
 at org.codehaus.jackson.map.ObjectMapper._readMapAndClose(ObjectMapper.java:2732)
 at org.codehaus.jackson.map.ObjectMapper.readValue(ObjectMapper.java:1973)
 at com.heroku.api.parser.JacksonParser.parse(JacksonParser.java:38)
 ... 84 common frames omitted`

Please, can You help cause it's blocker issue?

--- a/heroku-api/src/main/java/com/heroku/api/App.java
+++ b/heroku-api/src/main/java/com/heroku/api/App.java
@@ -22,8 +22,8 @@ public class App implements Serializable {
     String git_url;
     String buildpack_provided_description;
     String released_at;
-    int slug_size;
-    int repo_size;
+    long slug_size;
+    long repo_size;
     boolean maintenance;
     App.Space space;
 
@@ -97,11 +97,11 @@ public class App implements Serializable {
         this.buildpack_provided_description = buildpack_provided_description;
     }
 
-    private void setSlug_size(int slug_size) {
+    private void setSlug_size(long slug_size) {
         this.slug_size = slug_size;
     }
 
-    private void setRepo_size(int repo_size) {
+    private void setRepo_size(long repo_size) {
         this.repo_size = repo_size;
     }
 
@@ -146,11 +146,11 @@ public class App implements Serializable {
         return requested_stack;
     }
 
-    public int getSlugSize() {
+    public long getSlugSize() {
         return slug_size;
     }
 
-    public int getRepoSize() {
+    public long getRepoSize() {
         return repo_size;
     }
 

